psalm.xml

```
@@ -1,19 +1,22 @@
 <?xml version="1.0"?>
 <psalm
-	errorLevel="6"
+	errorLevel="4"
 	resolveFromConfigFile="true"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xmlns="https://getpsalm.org/schema/config"
 	xsi:schemaLocation="https://getpsalm.org/schema/config"
 	findUnusedBaselineEntry="false"
 	findUnusedCode="false"
+	findUnusedVariablesAndParams="true"
 	autoloader="bootstrap.php"
 >
 	<projectFiles>
 		<directory name="lib" />
 		<directory name="tests" />
 		<ignoreFiles>
 			<directory name="vendor" />
+			<directory name="../../**" />
```

```suggestion
			<directory name="../../**" />
```

it simply ignore error from server code error like

```error
Error: server/lib/public/BackgroundJob/TimedJob.php:92:32: MoreSpecificImplementedParamType: Argument 1 of OCP\BackgroundJob\TimedJob::execute has the more specific type 'OC\BackgroundJob\JobList', expecting 'OCP\BackgroundJob\IJobList' as defined by OCP\BackgroundJob\IJob::execute (see https://psalm.dev/140)
------------------------------
1 errors found
```